lib/kontrast/global_runner.rb

```diff
@@ -0,0 +1,121 @@
+require "yaml"
+require "net/http"
+
+module Kontrast
+    class GlobalRunner
```

Comment:
I think GlobalRunner is fine actually, `AbstractRunner` is the one that needs to change.